packages/graphql-language-service-interface/src/GraphQLLanguageService.js

```diff
@@ -103,10 +98,7 @@ export class GraphQLLanguageService {
     const appName = this._graphQLConfig.getAppConfigNameByFilePath(filePath);
```

Comment:
Good catch! Fixed.
Probably there is need to add some test-case that covers this piece of code